src/int/specialized_div_rem/mod.rs

```diff
@@ -77,8 +74,22 @@ const USE_LZ: bool = if cfg!(target_feature = "thumb-mode") {
     cfg!(target_feature = "v5te")
 };
 
-// All other targets Rust supports have CLZ instructions
-#[cfg(not(any(target_arch = "arm", target_arch = "riscv32", target_arch = "riscv64")))]
+// LZD or LZCNT on SPARC only exists for the VIS 3 extension and later.
+#[cfg(any(target_arch = "sparc", target_arch = "sparc64"))]
+const USE_LZ: bool = cfg!(target_feature = "vis3");
+
+// The `B` extension on RISC-V determines if a CLZ assembly instruction exists
+#[cfg(any(target_arch = "riscv32", target_arch = "riscv64"))]
+const USE_LZ: bool = cfg!(target_feature = "b");
+
+// All other common targets Rust supports should have CLZ instructions
+#[cfg(not(any(
+    target_arch = "arm",
+    target_arch = "sparc",
+    target_arch = "sparc64",
+    target_arch = "riscv32",
+    target_arch = "riscv64"
+)))]
```

Comment:
Instead of conditional definitions of this constant could it use `||` and `cfg!`?

Reply:
good idea